src/Mod/Path/PathScripts/PathSurface.py

```diff
@@ -23,33 +23,41 @@
 # ***************************************************************************
 # *                                                                         *
 # *   Additional modifications and contributions beginning 2019             *
-# *   by Russell Johnson  <russ4262@gmail.com>  2020-03-18 12:29 CST        *
+# *   by Russell Johnson  <russ4262@gmail.com>  2020-04-09 16:55 CST        *
 # *                                                                         *
 # ***************************************************************************
 
 from __future__ import print_function
 
+__title__ = "Path Surface Operation"
+__author__ = "sliptonic (Brad Collette)"
+__url__ = "http://www.freecadweb.org"
+__doc__ = "Class and implementation of Mill Facing operation."
+__contributors__ = "russ4262 (Russell Johnson)"
+
+# OCL must be installed
+try:
+    import ocl
+except ImportError:
+    FreeCAD.Console.PrintError(
+        translate("PathSurface", "This operation requires OpenCamLib to be installed.") + "\n")
+    import sys
+    sys.exit(translate("PathSurface", "This operation requires OpenCamLib to be installed."))
```

Comment:
I'm not sure `sys.exit` is good here. If I'm not mistaken this terminates the entire Python interpreter, so maybe it closes the entire FreeCAD application.

Reply:
Fixed.
Swapped it for a `raise` statement.